faq/prebid-server-faq.md

```diff
@@ -75,20 +75,98 @@ Syncing in the AMP scenario uses the [load-cookie.html](/dev-docs/show-prebid-ad
 creates or updates the `uids` cookie.
 
-The most common source of requests for Prebid Server is from Prebid.js:
-
-0) Assume that the user doesn't have any cookies for the Prebid Server domain.
-1) User loads a page with Prebid.js that's going to call Prebid Server -- i.e. the pub has set up s2sConfig.
-2) Immediately after seeing that s2sConfig is setup, Prebid.js calls Prebid Server's `/cookie-sync` endpoint to initiate syncing
-3) Prebid Server sees there no `uids` cookie, so responds to the browser with a list of pixel syncs for bidders that need to be synced.
-4) Prebid.js places all of the pixels on the page, but in the meantime, also initiates the auction.
-5) Because the syncs haven't completed yet, the auction call to Prebid Server doesn't yet contain the uids cookie.
-6) The first auction happens without IDs
-7) At some point later, the pixels come back to Prebid Server through a /setuid redirect, setting (or updating) the `uids` cookie.
-8) The second page view will have the IDs available.
-
-There's a nuance here: the company that's hosting Prebid Server can configure it to read and utilize their exchange's 
+The most common source of requests for Prebid Server is from Prebid.js in a scenario where the user doesn't have any cookies for the Prebid Server domain.
+1. The user loads a page with Prebid.js that's going to call Prebid Server -- i.e. the pub has set up s2sConfig.
+2. Immediately after confirming that s2sConfig is setup, Prebid.js calls Prebid Server's /cookie-sync endpoint to initiate syncing
+3. Prebid Server determines there is no `uids` cookie and responds to the browser with a list of pixel syncs for bidders that need to be synced.
+4. Prebid.js places all of the pixels on the page and initiates the auction.
+5. Because the syncs haven't completed, the auction call to Prebid Server will not contain the uids cookie.
+6. The first auction occurs without IDs
+7. At some point later, the pixels come back to Prebid Server through a /setuid redirect, setting (or updating) the uids cookie.
+8. The second page view will have the IDs available.
+
+
+
+{: .alert.alert-info :}
+Note: the company that's hosting Prebid Server can configure it to read and utilize their exchange's 
 native cookie. i.e. if you're using Rubicon Project's Prebid Server, it can read their 'khaos' cookie, and if you're using 
 AppNexus' Prebid Server, it can read their 'uuid2' cookie. In other words, if the host company is an exchange and the user 
 has the exchange cookie, the host company will have an ID one page-view sooner than the other bidders. This gives a slight edge to
 the hosting company in some scenarios, but it's technically unavoidable and better for both buyers and sellers to have one ID available rather than zero.
+
+## How does Prebid Server support privay signals?
+
+### Mobile 'Limit Ad Tracking' flag
+
+If PBS receives 'device.lmt' flag in the OpenRTB request, it does the following anonymization:
```

Comment:
Doesn't look like PBS-Go supports this. Added as a high priority item in our backlog.